src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -510,15 +512,17 @@ export const useSwapper = () => {
   )
 
   const updateQuote = useCallback(
-    async ({
-      amount,
-      sellAsset,
-      buyAsset,
-      feeAsset,
-      action,
-      forceQuote,
-      selectedCurrencyToUsdRate,
-    }: GetQuoteInput) => {
+    async (args: GetQuoteInput) => {
+      _getQuoteArgs = args
```

Comment:
Store the args used for the last quote so they can be reused by `refreshQuote()`.